Update Splits and KeyValue components to use flexbox instead of tables

wooferzfg opened the pull request:
Fix #241 

We'll still need to do another refactor to get two row mode to work for the Splits, but it should be significantly easier to get to that point from here.

CryZe commented:
Some weird stuff seems to be happening with the first split here:

![https://i.imgur.com/KChUxKM.gif](https://i.imgur.com/KChUxKM.gif)

CryZe commented:
Also the value seems to not be vertically aligned properly. Not sure if that changed or always was like this:

![https://i.imgur.com/GElp1PJ.png](https://i.imgur.com/GElp1PJ.png)

wooferzfg commented:
@CryZe Those issues should be resolved now.
- I added a 1 pixel top padding to split names in the Splits component and keys in the KeyValue component to address the alignment issue.
- We now ignore any empty colums that are at the end of a split's columns array.
- I fixed a regression on Firefox for the width of the control buttons.
